docs/quickstart.md

```diff
@@ -9,7 +9,8 @@ Go to [playground.rebuff.ai](https://playground.rebuff.ai) and get your API key
 ```python
 from rebuff import Rebuff
```

Comment:
Optional: We could update the `playground.rebuff.ai` link above to go directly to `https://rebuff.ai/playground` as `playground.rebuff.ai` is now a redirect.